protocol/signalfx/signalfxforwarder.go

```diff
@@ -112,6 +112,17 @@ func NewSignalfxJSONForwarer(url string, timeout time.Duration,
 		},
 		TLSHandshakeTimeout: timeout,
 	}
+
+	req, _ := http.NewRequest("POST", url, nil)
+	if req != nil {
+		purl, err := tr.Proxy(req)
```

Comment:
If you want to test this, create a function similar to this:

```
func logProxyURL(t *http.Transport) (wasLogged bool) {
// Put your code here
}
```

You can then call that with a transport that has a Proxy that returns false, and another that returns true, and test if wasLogged.